src/05-partial-mocking/index.test.ts

```diff
@@ -11,10 +19,15 @@ describe('partial mocking', () => {
   });
 
   test('mockOne, mockTwo, mockThree should not log into console', () => {
-    // Write your test here
+    expect(mockOne()).toBe(null);
```

Comment:
Test says "should not log into console". Assertions do not verify that fact.
`console.log` should be mocked as in the 2nd test and checked if it was called.